composer.json

```diff
@@ -10,11 +10,12 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
-        "ezsystems/ezpublish-kernel": "~6.7.8 || ~6.13.4 || ^7.0",
+        "php": ">=7.1",
+        "ezsystems/ezpublish-kernel": "^7.4@dev",
```

Comment:
Please do these kernel and php changes post merge, half the point doing this from support side it to do a 0.2.x for v2 (2.0-2.4) users letting them know if something is wrong with their install _(including which version they are on and if it is expired)_

Reply:
Then I think also `\EzSystems\EzSupportToolsBundle\AdminUi\Component\EzInfoTwigComponent` should go to higher version, as it creates dependency on adminUI (and php7+).